frontend/app/templates/components/footer-main.hbs

```diff
@@ -0,0 +1,47 @@
+<div class="ui center aligned container">
+  <div class="ui stackable inverted divided grid">
+    <div class="three wide column">
+      <div class="ui inverted link list">
+        <strong class="item">Use Badgeyay</strong>
+        <a class="item" href="#" target="_blank">Sitemap</a>
+      </div>
+    </div>
+    <div class="three wide column">
+      <div class="ui inverted link list">
+        <strong class="item">Categories</strong>
+      </div>
+    </div>
+    <div class="three wide column">
+      <div class="ui inverted link list">
+        <strong class="item">Locations</strong>
```

Comment:
What is the significance of `Locations` here?

Reply:
These are the locations at which we can ship the badges :+1:

Reply:
Why these locations? when we can ship globally. A user has to just generate PDF through our service and send those PDF's for printing.

Reply:
@gabru-md @ParthS007 If that's the case then `Location` is misleading title. It should be `Shipping Locations`